lib/ace/ext/papeeria/spellchecker.coffee

```diff
@@ -1,39 +1,82 @@
 define( ->
-  # This will be removed and replaced with real data from server.
-  # BTW, this is kinda template for data to send from server.
-  TEST_JSON_TYPOS = {
-    "blablabla": [
-      "blah-blah-blah"
-      "blabber"
-      "blabbed"
-      "salable"
-    ]
-  }
 
+  STATE_COMPLETE = 4
+
+
+  getJson = (url, data, onSuccess) ->
+    xhr = new XMLHttpRequest()
```

Comment:
What about [`jQuery.getJSON`](http://api.jquery.com/jquery.getjson/)? jQuery is used all over ACE code, and we use `getJSON` e.g. in `tex_completer`